1-exercises/B-setting-values/exercise2.js

```diff
@@ -25,7 +25,9 @@ let student = {
   - Use bracket notation to change the value of hasPassed
 */
 
-// write code here
+if (student["attendance"]>= 90 && student["examScore"]) {
+  student.hasPassed = true;
```

Comment:
I'm not sure if line 29 is bracket notation? how else could you write this line?